plottr/gui/data_display.py

```diff
@@ -32,15 +32,13 @@ def __init__(self, parent: Optional[QtWidgets.QWidget] = None,
 
     def _makeItem(self, name: str) -> QtWidgets.QTreeWidgetItem:
         shape = self._dataShapes.get(name, tuple())
-        unit = self._dataStructure[name].get('unit', '')
-        label = f"{name} [{unit}]"
+        label = f"{name} [{self._dataStructure.label(name)}]"
         deps = "("
         for i, d in enumerate(self._dataStructure.axes(name)):
             if i > 0:
                 deps += ", "
-            axunit = self._dataStructure[d].get('unit', '')
-            axlabel = f"{d} [{axunit}]"
-            deps += axlabel
+            axlabel = self._dataStructure.label(d)
```

Comment:
so units are not welcome in the axis label?

Reply:
@astafan8 This is not the axis label used in the plot but the dataset information in inspectr

Reply:
oh, ok, thanks)

Reply:
units are still captured here. I simply change it to use the internal label function, which creates the label + unit string, rather than generating that string locally.